src/lib/util/Cache/MemberTags.ts

```diff
@@ -118,6 +130,14 @@ export class MemberTags extends Collection<string, MemberTag> {
 		return highest;
 	}
 
+	private requestHandlerGet(id: string) {
+		return this.guild.members.fetch(id);
+	}
+
+	private requestHandlerGetAll(ids: readonly string[]) {
+		return Promise.all(ids.map(id => this.guild.members.fetch(id)));
```

Comment:
After https://github.com/discordjs/discord.js/pull/3562

```suggestion
		return this.guild.members.fetch({ user: ids });
```

Reply:
I added a note in 94e68e4, thanks!